indico/web/client/js/jquery/extensions/global.js

```diff
@@ -247,19 +247,26 @@ $(document).ready(function() {
   showFormErrors();
 
   // Show form creation dialog if hash is present in the URL
-  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d+))?$/);
+  let match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*)(?::([a-f0-9-]{36}))?)?$/);
   if (match) {
-    var eventType = match[1];
-    var categoryId = match[2];
-    var title = {
+    let eventType = match[1];
+    let categoryId = match[2];
+    let eventUUID = match[3];
+    let title = {
       lecture: $T.gettext('Create new lecture'),
       meeting: $T.gettext('Create new meeting'),
       conference: $T.gettext('Create new conference'),
     }[eventType];
-    var url =
-      match[2] !== undefined
-        ? build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId})
-        : build_url(Indico.Urls.EventCreation, {event_type: eventType});
+    let url;
+    if (match[2]) {
+      url = build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId});
+    } else if (match[3]) {
+      url = build_url(Indico.Urls.EventCreation, {event_type: eventType, event_uuid: eventUUID});
+    } else if (match[2] && match[3]) {
+      // TODO: When both categoryId and eventUUID are present.
```

Comment:
I think when we discussed this the last time we said we'd pass both params to `build_url`. It most likely works out of the box, even if nothing generates such URLs yet. You can easily test it yourself by opening `#create-event:meeting:1:<uuid>` in which case the dialog should have category 1 preselected.

Reply:
Didn't realize it would work as is. Updated.